src/utils.js

```diff
@@ -72,6 +72,21 @@ function esriTypeMap (type) {
       return 'esriFieldTypeInteger'
     case 'date':
       return 'esriFieldTypeDate'
+    case 'esrifieldtypeblob':
+    case 'esrifieldtypedate':
+    case 'esrifieldtypedouble':
+    case 'esrifieldtypegeometry':
+    case 'esrifieldtypeglobalid':
+    case 'esrifieldtypeguid':
+    case 'esrifieldtypeinteger':
+    case 'esrifieldtypeoid':
```

Comment:
I think we should follow that pattern we set where we remove `esriFieldType`

Reply:
Is this a pattern currently found in FeatureServer? looking but don't see it

Reply:
it's in this very function.
Date => EsriFieldTypeDate
String => EsriFieldTypeString

Reply:
Ok, misunderstood.  We should expect that the provider's metadata `fields` will omit the `esriFieldType` prefix, so we should omit it here as well.

Reply:
right. when we counsel people on how to use metadata, we should tell them to list fields without the `esriType` as this metadata might be used by other output types.